app/api/v1/views.py

```diff
@@ -120,11 +120,11 @@ def get(self, id):
         return make_response(
             jsonify(
                 {
-                    "status": 200,
+                    "status": 404,
```

Comment:
why do you return two status codes here?

Reply:
I'm resolving that immediately.